MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentBackground       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of segment when it is disabled. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentTextColor        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment item text when it is disabled.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionBackground         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of segment when it is selected.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionTextColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of text when it is selected. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionStroke           
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of border when it is selected.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlHoveredBackground            
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of segment when it is hovered.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlKeyboardFocusStroke             
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of border when it is keyboard pressed.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalStrokeThickness              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Thickness of the stroke in the in segmented control.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalCornerRadius               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of corner radius in segmented control.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalSegmentCornerRadius                
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of segment corner radius.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalFontSize                 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of text in segmented control.   
```

Comment:
Font size of the segment item in segmented control.

Reply:
Updated